components/bm1397/bm1366.c

```diff
@@ -540,6 +556,49 @@ static void _send_read_address(void)
     _send_BM1366((TYPE_CMD | GROUP_ALL | CMD_READ), read_address, 2, false);
 }
 
+static int _calculate_chip_number(unsigned int actual_chip_count)
+{
+    int i = 0;
+    if(actual_chip_count == 1)
```

Comment:
the algo for this function may be written using interger algebra instead of this if else if else if structure.

I don't really understand the final usage of it, but as it is only for the added test (will not be used by actual mining), why not.